chirp/drivers/ft1d.py

```diff
@@ -204,17 +205,29 @@
      autostep:1,
      automode:1,
      unknown9:3;
-} memory[%d];
+};
 
-#seekto 0x280A;
-struct {
+#seekto 0x2D4A;
+struct  memslot  memory[%d];
```

Comment:
So if I may make a suggestion here. Instead of using these anonymous position-based format specifiers, it would be nice to use named ones. So this line would be:
```
struct memslot memory[%(max_mem)d];
```
and then your code below can be something like:
```
mem_params = {
    'max_mem': 900,
    'max_flag': 900,
    'dtmf_loc': 0xE4A,
}
mem_format % self._mem_params
```
Doing that will be much easier to reason about and debug...

Reply:
Great idea. It'll require change to the FTM-3200D driver, but darn well worth it. Indeed, for the time being I still define that max_flag, but don't use it; I can use max_mem in each case, rather than duplicate the positional parameter.